examples/jsonplaceholder.graphql

```diff
@@ -1,28 +1,281 @@
+# Define the schema with upstream configuration and server settings
 schema
+  # Specify server configuration: Start GraphQL server at 0.0.0.0:8000
   @server(port: 8000)
-  @upstream(baseURL: "http://jsonplaceholder.typicode.com", httpCache: 42, batch: {delay: 100}) {
+  # Specify a base URL for all HTTP requests
+  @upstream(baseURL: "https://jsonplaceholder.typicode.com") {
   query: Query
+  mutation: Mutation
 }
 
+# Define the Query type with various fields
 type Query {
-  posts: [Post] @http(path: "/posts")
-  users: [User] @http(path: "/users")
+  # Fetch a post by ID
+  post(id: Int!): Post @http(path: "/posts/{{.args.id}}")
+  # Fetch all posts by a specific user
+  posts(user: Int!): [Post] @http(path: "/posts?userId={{.args.user}}")
```

Comment:
```suggestion
  posts(userId: Int): [Post] @http(path: "/posts?userId={{.args.user}}")
```
User should be optional. This is true for all the types that return a list of values. Please update the arguments and drop required.